src/misc/AttachShadowPolyfill.ts

```diff
@@ -4,24 +4,34 @@ import 'styling/_AttachShadowPolyfill';
 export interface IShadowOptions {
   title: string;
   onSizeChanged?: Function;
+  useIFrame?: Boolean;
```

Comment:
Should the `f` be lowercase?

Reply:
I always thought it was an iFrame ?

`use-i-frame` or `use-iframe` in the html option?
I feel like people might mistype the 2nd one and try to do `use-frame` instead? What do you think?

Reply:
honestly, the second looks really natural to me, _but_ I see that the typescript is `HTMLIFrameElement`.
The current casing is consistent with the type, so leaving it as-is makes sense 👍